packages/node_modules/express-pouchdb/package.json

```diff
@@ -10,5 +10,39 @@
   "main": "./lib/index.js",
   "files": [
     "lib"
-  ]
+  ],
+  "version": "2.3.4",
+  "dependencies": {
+    "basic-auth": "^1.1.0",
+    "body-parser": "^1.16.1",
+    "compression": "^1.6.2",
+    "cookie-parser": "^1.4.3",
+    "denodeify": "^1.2.1",
+    "express": "^4.10.4",
+    "extend": "^3.0.0",
+    "header-case-normalizer": "^1.0.3",
+    "mkdirp": "^0.5.0",
+    "multiparty": "^4.1.3",
+    "on-finished": "^2.3.0",
+    "pouchdb-all-dbs": "^1.0.2",
+    "pouchdb-auth": "^2.1.1",
+    "pouchdb-collections": "^6.1.2",
+    "pouchdb-find": "^0.10.4",
+    "pouchdb-list": "^1.1.0",
+    "pouchdb-promise": "^6.1.2",
+    "pouchdb-replicator": "^2.1.3",
+    "pouchdb-rewrite": "^1.0.7",
+    "pouchdb-security": "^1.2.6",
+    "pouchdb-show": "^1.0.8",
+    "pouchdb-size": "^1.2.2",
+    "pouchdb-update": "^1.0.8",
+    "pouchdb-validation": "^1.2.1",
+    "pouchdb-vhost": "^1.0.2",
+    "pouchdb-wrappers": "^1.3.6",
+    "raw-body": "^2.2.0",
+    "sanitize-filename": "^1.6.1",
+    "uuid": "^3.0.1",
+    "pouchdb-fauxton": "^0.0.5"
+  },
```

Comment:
@garrensmith you actually don't need to add any `dependencies` in the sub-packages; just keep it empty. They'll automatically be filled in by `release.sh`.